include/maxscale/config.hh

```diff
@@ -218,6 +219,7 @@ private:
     static config::ParamString              s_admin_pam_rw_service;
     static config::ParamString              s_admin_pam_ro_service;
     static config::ParamString              s_admin_ssl_key;
+    static config::ParamString              s_admin_ssl_cipher;
```

Comment:
Change this to `config::ParamEnum<mxb::ssl_version::Version>` so that unknown values are rejected.